pytorch_pfn_extras/reporting.py

```diff
@@ -367,6 +367,14 @@ def load_state_dict(self, to_load: Dict[str, Any]) -> None:
         self._x2 = float(_nograd(to_load['_x2']))
         self._n = int(_nograd(to_load['_n']))
 
+    def __add__(self, other: "Summary") -> "Summary":
+        s = Summary()
+        s._x = self._x + other._x
+        s._x2 = self._x2 + other._x2
+        s._n = self._n + other._n
+        s._deferred = self._deferred + other._deferred
```

Comment:
I am not sure if this will work, we should probably synchronize all the deferred results before merging summaries.